.github/workflows/chromatic.noyml

```diff
@@ -0,0 +1,41 @@
+name: Chromatic
```

Comment:
This has an odd extension. should it be `.yml`?

Reply:
Yes, it should.  I disabled it temporarily for testing.  For some reason the CI actions won't run at all on this PR anymore when I push.  I'm not sure what's going on.  I'm trying a few things like disabling this one workflow.